Diary/src/main/java/com/microsoft/device/display/samples/diary/FileOps.kt

```diff
@@ -0,0 +1,70 @@
+package com.microsoft.device.display.samples.diary
+
+import android.Manifest
+import android.content.Context
+import android.content.pm.PackageManager
+import android.os.Build
+import android.os.Environment
+import android.util.Log
+import androidx.core.content.ContextCompat
+import java.io.BufferedReader
+import java.io.File
+import java.io.FileReader
+import java.io.FileWriter
+import java.io.IOException
+
+class FileOps {
```

Comment:
It looks like there aren't fields stored in this class so you can just delete the class wrapper and leave the methods in the file.

That way, you don't have to instantiate a FileOps() instance and you can just directly call the methods in your code!